h/claim/__init__.py

```diff
@@ -0,0 +1,13 @@
+from webob.cookies import SignedSerializer
+
+
+def includeme(config):
+    config.include('.views')
+    config.include('.util')
+
+    # We use the shared session secret, but salt it with the namespace
+    # 'h.claim' -- only messages serialized with this salt will
+    # authenticate on deserialization.
+    secret = config.registry.settings['session.secret']
+    serializer = SignedSerializer(secret, 'h.claim')
+    config.registry.claim_serializer = serializer
```

Comment:
> I'm just being paranoid and raising the questions, even if I don't have the answers.

Same here. I would like to know a bit more about the background, so that I can be at peace with this.